### DISCUSSION TITLE
Hybrid code's ability to handle charge separation of the ions and electrons

### DISCUSSION THREAD

**Discussion by Aquios7** [BLAST-WarpX/warpx#5020] [Q&A] [answered]
I'm injecting a plasma beam into a high density plasma and would like to use hybrid for computational efficiency, as well as large simulation volumes. Would the hybrid code allow for a separation of ions and electrons once the beam enters the plasma, or is it not possible to have an accumulation of charge, i.e., separation of the beam particles in the hybrid calculation?

I also posted another question having to do with this simulation earlier. I thought it would be better to post my questions separately.

**Comment by roelof-groenewald** [comment, marked as answer]
Hi @Aquios7. Thanks for your question and sorry for the slow reply. By "hybrid" do you mean the Ohm's law based solver that uses fluid electrons? If so, that model cannot capture charge separation effects since it assumes the electron and ion density is equal.
